tools/uart_hw_test/README.md

```diff
@@ -104,3 +117,67 @@ Test Scenario 3: UART test with background iperf
 ######################################################################
 ################## UART Test End [PASS: 40, FAIL: 0] #################
 ```
+
+Below is a sample set of results for windows case :
```

Comment:
Sequence looks not good. Linux and Windows are mixed. It's not good to see.
Let's split them
```
## how to use in Linux

result sample for linux


## how to use in windows

result sample for windows
```

Reply:
@sunghan-chang what is your opinion on adding sample test-case result to README?
I would suggest we should only add final result line in README
"################## UART Test End [PASS: 40, FAIL: 0] #################"
This tells the developer what to expect if every UTC runs perfectly, which should be enough.

I feel that the complete test case output should be added as a comment in the PR.